catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliAttributeMap.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.nsili.common;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import ddf.catalog.data.Metacard;
+
+public class NsiliAttributeMap {
+    private static Map<String, String> nsiliToDdfMap = new HashMap<>();
+    private static Map<String, List<String>> ddfToNsiliMap = new HashMap<>();
+
+    static {
+        //Build up attribute mappings
+
+        nsiliToDdfMap.put("NSIL_APPROVAL.approvedBy", "approvedBy");
```

Comment:
Good catch. That's a good idea. I caught the Metacard side and completely missed the NSIL side.